### PR TITLE
fix data modification during grub_installdevice write

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: lib/cfa/grub2/install_device.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@ def self.parse(string)
 
       # gets list of devices and create file content from it
       def self.serialize(data)
+        # do not modify original data as serialize is not end of world
+        data = data.dup
```
Comment (lslezak) [inline lib/cfa/grub2/install_device.rb]: Actually I do not like much shadowing the arguments by local variable, what about renaming the argument? 

```ruby
  def self.serialize(input_data)
    data = input_data.dup
    ...
```

Comment (jreidinger) [inline lib/cfa/grub2/install_device.rb]: actually goal of this code is to shadow argument to avoid its modification.

Comment (lslezak) [inline lib/cfa/grub2/install_device.rb]: OK, I see...

**Comment by lslezak** [comment]
LGTM

**Pull request merged**